hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupInfoManagerImpl.java

```diff
@@ -726,9 +726,14 @@ private void updateCacheOfRSGroups(final Set<String> currentGroups) {
 
   // Called by ServerEventsListenerThread. Presume it has lock on this manager when it runs.
   private SortedSet<Address> getDefaultServers() {
+    return getDefaultServers(listRSGroups()/* get from rsGroupMap */);
+  }
+
+  // Called by ServerEventsListenerThread. Presume it has lock on this manager when it runs.
```

Comment:
Also pls leave this comment. The ServerEventsListenerThread calls the above method only.

Reply:
right, comment is not relevant on getDefaultServers(List<RSGroupInfo> rsGroupInfoList)  method.
but this PR is already merged. shall  i create new PR